src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id: number,
```

Comment:
Consider using semicolons instead of commas to separate properties in the interface for better consistency with TypeScript coding standards.

Reply:
thank you
as TS is pretty new for me I was not attentive to these moments
an did not payed attention to this conventions
they (TS tools) are flexy
but I agree - its more adequate to write `interface` in similar to `type` way)